conf.d/openstack.yaml.example

```diff
@@ -27,6 +27,9 @@ init_config:
       # need to set to false when using self-signed certs
       # ssl_verify: true
 
+     # Whether the dd-agent proxy should be used for openstack API requests (if set)
```

Comment:
Speaking as a newbie, what about adding `also`? Like `... should be also used for openstack ...`